firstvoices/backend/views/category_views.py

```diff
@@ -39,77 +45,90 @@
                     OpenApiExample(
                         "WORD|PHRASE",
                         value="WORD|PHRASE",
-                        description="Contains both. Order is not relevant here.",
+                        description=_(
+                            "Contains any of the specified types. Order is not relevant."
+                        ),
                     ),
                 ],
             )
         ],
     ),
     retrieve=extend_schema(
-        description="Details about a specific category.",
+        description=_("Details about a specific category."),
+        responses={
+            200: OpenApiResponse(
+                description=doc_strings.success_200_detail,
+                response=CategoryDetailSerializer,
+            ),
+            403: OpenApiResponse(description=doc_strings.error_403),
+            404: OpenApiResponse(description=doc_strings.error_404),
+        },
+    ),
+    destroy=extend_schema(
+        description=_("Delete a category."),
         responses={
-            200: CategoryDetailSerializer,
-            403: OpenApiResponse(description="Todo: Error Not Authorized"),
-            404: OpenApiResponse(description="Todo: Not Found"),
+            204: OpenApiResponse(
+                description=doc_strings.success_204_deleted,
+                response=CategoryDetailSerializer,
+            ),
+            400: OpenApiResponse(description=doc_strings.error_400_validation),
+            403: OpenApiResponse(description=doc_strings.error_403),
+            404: OpenApiResponse(description=doc_strings.error_404_missing_site),
         },
     ),
 )
 class CategoryViewSet(FVPermissionViewSetMixin, SiteContentViewSetMixin, ModelViewSet):
-    http_method_names = ["get"]
+    http_method_names = ["get", "delete"]
     valid_inputs = TypeOfDictionaryEntry.values
 
     def get_detail_queryset(self):
         site = self.get_validated_site()
-        if site.count() > 0:
```

Comment:
The rest of the changes in this view are mostly indentation, sorry! I removed the conditionals about the site count, since get_validated_site will raise an exception if the queryset is empty